packages/office-ui-fabric-react/src/components/Persona/Persona.test.tsx

```diff
@@ -23,10 +23,10 @@ const STYLES = {
  */
 export const wrapPersona = (
   example: IPersonaSharedProps,
-  isCoin: boolean = false
+  isOnRenderCoinCB: boolean = false
```

Comment:
Would prefer typing out CB => callback.
Even better, what about shouldWrapPersonaCoin? That explains exactly what is happening